client/my-sites/importer/importer-action-buttons/close-button.jsx

```diff
@@ -35,15 +35,16 @@ export class ImporterCloseButton extends React.PureComponent {
 
 	handleClick = () => {
 		const {
-			importerStatus: { importerId, type },
+			importerStatus: { engine, importerId },
 			site: { ID: siteId },
 		} = this.props;
 
 		cancelImport( siteId, importerId );
 
 		this.props.recordTracksEvent( 'calypso_importer_main_cancel_clicked', {
 			blog_id: siteId,
-			importer_id: type,
+			engine,
```

Comment:
Does this confuse analytics analysis if we have a prop that's not there for previous events?

Reply:
I don't believe so -- but I'm happy to just remove engine altogether, as it's completely redundant.